packages/check-html-links/src/validateFolder.js

```diff
@@ -312,6 +353,7 @@ export async function validateFiles(files, rootDir, opts) {
     await resolveLinks(links, { htmlFilePath, rootDir, ignoreUsage });
   }
   await validateLocalFiles(checkLocalFiles);
+  await validateExternalLinks(checkExternalLinks);
```

Comment:
can we disable this by default and add a flag to enable it?

e.g. something like
```
check-html-links --validate-external
```

that way it's fast by default but you can do a full check if you want